spyql/processor.py

```diff
@@ -129,6 +129,27 @@ def handle_1st_data_row(self, row):
         # dict of {col1: value1, ...}
         self.translations["row"] = f"NullSafeDict(zip(_names, {cols_expr}))"
 
+    def expand_col_names(self, all_col_names, name):
+        """
+        Expands column wildcard operator and modifiers
+        """
+        except_pattern = re.compile(r"\*\s*EXCEPT\s*\((.*)\)", re.IGNORECASE)
+        except_match = re.search(except_pattern, name)
+
+        if name == "*":
+            return all_col_names
+        elif except_match:
+            except_col_names = [
+                col_name.strip() for col_name in except_match.group(1).split(",")
+            ]
+            return [
```

Comment:
We might have to call `strings.put_strings_back(col_name)`, in case the column to exclude is quoted. 
Again, I would to this in the parsing stage to avoid this kind of dirty details in the processor.